### PR TITLE
Update README Applications

### PR DESCRIPTION
**Pull request by willemdek11**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,29 +35,26 @@ to serve the site. Now open `http://localhost:4000` in your browser to preview y
...
+---
+layout: application
+title: "Deep- and Machine- Learning"
+subtitle: "JUNIPER"
+icon: stethoscope
+author: "Andrew Stubbs"
+date: 2020-01-28 23:45:13 -0400
+background: '/img/bg-index.jpg'
+---
+<p>Never in all their history have men been able truly to conceive of the world as one: a single sphere, a globe, having the qualities of a globe, a round earth in which all the directions eventually meet, in which there is no center because every point, or none, is center — an equal earth which all men occupy as equals. The airman's earth, if free men make it, will be truly round: a globe in practice, not in theory.</p>
```
Comment (shiltemann) [inline README.md:53]: do we need the <p> tags or?

Comment (willemdek11) [reply README.md:53]: You don't need it when you have only one paragraph, but if you have multiple you have to give it in HTML format to give it the right format. So I would say, yes you need them.

Comment (shiltemann) [reply README.md:54]: and if you leave empty line between paragraphs? it should be converting those properly.. if not we should check why not, shouldn't be something for content writers to worry about, weird..

**Review by shiltemann — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (hexylena) [reply README.md:54]: @shiltemann is right, just needs double empty line to be proper html

Comment (willemdek11) [reply README.md]: Yep it works 👍🏻

**Pull request merged**